### DISCUSSION TITLE
Using GItHub for Manual test case management

### DISCUSSION THREAD

**Discussion by Jyothi-Pataje** [community/community#121694] [Projects and Issues]
### Hi everyone,

We would like to migrate from Azure devOps to to GitHub for project management. Currently we use Git only for code manage but we would like to extend this to Project management as well. So as part of that need to write test cases in Git and want to maintain by linking it to the ticket created in it. Can you please guide us can this be achievable?

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @Jyothi-Pataje , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Comment by Jyothi-Pataje** [comment]
[like]  Pataje, Jyothi P SBOBNG-PTIY/DIJ reacted to your message:
________________________________
